dev/archery/archery/crossbow/reports.py

```diff
@@ -121,6 +121,61 @@ def show(self, outstream, asset_callback=None):
                                    asset))
 
 
+class JsonReport(Report):
+
+    HEADER = textwrap.dedent("""
```

Comment:
Do we really need `HEADER`, `STATUS_HEADERS` and `TASKS` given that we are not formatting any text here? I guess those are copy/paste from the email reporter that had to format the mail